Create modern-language-association-for-JEI.csl

SirohiKumar opened the pull request:
The Journal of Emerging Investigators (JEI: https://emerginginvestigators.org/submissions/references) requires a version of MLA citation with specialized in text references. This version of MLA format in Zotero will help students interested in submitting to JEI.

bwiernik commented:
Hi there! Thanks for the submission. 

Please rename the style to `journal-of-emerging-investigators.csl` and change the style ID and self link accordingly   

Also please remove the `author-short` macro if it's not being used.

SirohiKumar commented:
Hi Brenton,

I renamed the style and removed the unused macro. 

Sirohi

--
Sirohi Kumar 
(she/her/hers)
Smith College - Class of 2026


> On Jul 8, 2023, at 3:39 PM, Brenton M. Wiernik ***@***.***> wrote:
> 
> 
> Hi there! Thanks for the submission.
> 
> Please rename the style to journal-of-emerging-investigators.csl and change the style ID and self link accordingly
> 
> Also please remove the author-short macro if it's not being used.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/citation-style-language/styles/pull/6593#issuecomment-1627475102>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AT63E4EJG2M656BVQJC3ONLXPGZQDANCNFSM6AAAAAA2CTC4HQ>.
> You are receiving this because you authored the thread.
>

SirohiKumar commented:
just removed the other unused macro, "title-short"
